backend/Recodo.DAL/Entities/Comment.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recodo.DAL.Entities
+{
+    public class Comment
+    {
+        public int Id { get; set; }
+        public string Body { get; set; }
+        public int AuthorId { get; set; }
+        public User Author { get; set; }
+        public int VideoId { get; set; }
+        public Video Video { get; set; }
```

Comment:
I'm not sure we need navigation properties everywhere, especially for entities like these

Reply:
@Jyuart  believe it's not bad, for example if we want to get avatar for user who added comment